msal/src/internal/java/com/microsoft/identity/client/MsalUtils.java

```diff
@@ -277,7 +279,7 @@ static String getChromePackage(final Context context) {
      * @return The Map of the items decoded with the given delimiter.
      */
     static Map<String, String> decodeUrlToMap(final String url, final String delimiter) {
-        final Map<String, String> decodedUrlMap = new HashMap<>();
+        final Map<String, String> decodedUrlMap = new LinkedHashMap<>();
```

Comment:
actually order doesn't matter when the request is sent over the wire, let me remove it.